src/main/java/models/blogPost.java

```diff
@@ -0,0 +1,70 @@
+package models;
+
+import java.time.LocalDateTime;
+import java.util.Set;
+
+public class blogPost {
+    private String blogId;
+    private String authorId;
+    private String title;
+    private String description;
+    private Set<String> categories;
```

Comment:
typically I initialize this -- e.g., private Set<String> categories = new HashSet<String>();  -- otherwise you're going to run into useless NPE errors trying to add things to this in the future.